tutteli-gradle-spek/src/main/groovy/ch/tutteli/gradle/plugins/spek/SpekPlugin.groovy

```diff
@@ -65,12 +66,32 @@ class SpekPlugin implements Plugin<Project> {
     }
 
     private static String getKotlinVersion(Project project) {
-        def plugins = project.plugins
-        def kotlinPlugin = plugins.findPlugin('kotlin')
-            ?: plugins.findPlugin('kotlin-platform-jvm')
-            ?: plugins.findPlugin('org.jetbrains.kotlin.multiplatform')
-            ?: plugins.findPlugin('org.jetbrains.kotlin.jvm')
-        def version = kotlinPlugin?.getKotlinPluginVersion()
+        def version
+        try {
+            version = KotlinPluginWrapperKt.getKotlinPluginVersion(project)
+        } catch (MissingMethodException e) {
+            e.printStackTrace()
```

Comment:
Please do not use `printStackTrace` as it will pollute the console output.

Reply:
thanks, that was only a test and should have been removed